assets/js/util.js

```diff
@@ -242,8 +243,29 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function ajaxCallForAllModes(methodType, request, endpoint, callbackSuccess, callbackError, requestName) {
+    requestName = $.ajax({
+        url: config.APY_URL + endpoint,
+        beforeSend: ajaxSend,
+        complete: function () {
+            ajaxComplete();
+            requestName = undefined;
+        },
+        data: request,
+        type: methodType,
+        contentType: 'application/x-www-form-urlencoded; charset=UTF-8',
+        dataType: 'json',
+        success: function (data) {
+            callbackSuccess(data);
+        },
```

Comment:
This can simply be `success: callbackSuccess` or even more elegantly: `success: success`. Instead of making each of these separate options passed through the arguments, you could make it accept an `options` Object and merge that Object with the one here. That would be the most extensible solution. Then the call sites for this function would provide their input as an object (`endpoint` can remain a separate argument).